test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+jest.mock('@folio/stripes/components', () => ({
+  Button: jest.fn(({
+    children,
+    onClick = jest.fn(),
+    // eslint-disable-next-line no-unused-vars
+    buttonStyle, buttonRef,
+    ...rest
+  }) => (
+    <button data-test-button type="button" {...rest} onClick={onClick}>
```

Comment:
Could you follow single style approach?

```suggestion
    <button
      data-test-button
      type="button"
      onClick={onClick}
       {...rest}
    >
```